src/Nncase.Core/Tensor.cs

```diff
@@ -370,6 +370,32 @@ public static Tensor<T> FromConst<T>(Const @const, CastMode castMode = CastMode.
       where T : unmanaged, IEquatable<T>
       => FromConst(@const).Cast<T>(castMode);
 
+    /// <summary>
+    /// Return a tensor of given shape and type, filled with zeros..
+    /// </summary>
+    /// <typeparam name="T">unmanaged type.</typeparam>
+    /// <param name="shape">shape.</param>
+    /// <returns>Tensor{T}.</returns>
+    public static Tensor Zeros<T>(Shape shape)
+        where T : unmanaged, IEquatable<T>
+    {
+        var value = (T)Convert.ChangeType(0, typeof(T));
+        return Tensor.FromValues<T>(shape, value);
+    }
+
+    /// <summary>
+    /// Return a tensor of given shape and type, filled with ones..
+    /// </summary>
+    /// <typeparam name="T">unmanaged type.</typeparam>
+    /// <param name="shape">shape.</param>
+    /// <returns>Tensor{T}.</returns>
+    public static Tensor Ones<T>(Shape shape)
+        where T : unmanaged, IEquatable<T>
+    {
+        var value = (T)Convert.ChangeType(1, typeof(T));
+        return Tensor.FromValues<T>(shape, value);
+    }
```

Comment:
- Should use `ReadOnlySpan<int> dimensions` instead of `Shape`.
- Should use `Tensor.FromScalar` instead of `Tensor.FromValues`.